src/world/besieged_system.cpp

```diff
@@ -20,3 +20,111 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 */
 
 #include "besieged_system.h"
+#include "common/mmo.h"
+#include "common/settings.h"
+#include "common/socket.h"
+#include "map/besieged_data.h"
+#include "world/message_server.h"
+
+#include <zmq.hpp>
```

Comment:
I'd like the ZMQ impl details to not leak out into other files, you can add a helper that turns a vector straight into an equivalent zmq message, and then send immediately

Reply:
zmq.hpp is still an internal zmq header, thus the angle brackets

Reply:
Done. Changed in conquest_system too.